src/vocab.rs

```diff
@@ -34,6 +35,39 @@ impl PyVocab {
             }
         })
     }
+
+    fn ngram_indices(&self, word: &str) -> PyResult<Vec<Py<PyTuple>>> {
+        let embeds = self.embeddings.borrow();
+        let gil = pyo3::Python::acquire_gil();
+        match embeds.vocab() {
+            VocabWrap::FastTextSubwordVocab(inner) => Ok(inner
+                .ngram_indices(word)
+                .unwrap()
+                .into_iter()
+                .map(|pair| pair.into_py(gil.python()))
+                .collect()),
+            VocabWrap::FinalfusionSubwordVocab(inner) => Ok(inner
+                .ngram_indices(word)
+                .unwrap()
+                .into_iter()
+                .map(|pair| pair.into_py(gil.python()))
+                .collect()),
+            _ => Err(exceptions::ValueError::py_err(
```

Comment:
Catch-all match arms can be quite hard to debug since the compiler doesn't complain when a new variant is added to the enum. We might end up adding another `VocabType` and we wouldn't be aware of all the places in the code where we need to handle things differently.
 So, when matching enums or something that could change from our side, we should always use explicit match arms. In this case, `VocabWrap::SimpleVocab(_)`.